axelrod/random_.py

```diff
@@ -12,3 +12,9 @@ def random_choice(p=0.5):
     if r < p:
         return Actions.C
     return Actions.D
+
+def randrange(a, b):
+    """Python 2 / 3 compatible randrange."""
```

Comment:
Could you write a bit more saying what this takes and returns (numpy syntax? If we're going to stick to that.).